File: app/src/main/java/com/ninhttd/devtest/presentation/product/adapter/SpCungLoaiAdapter.java
package com.ninhttd.devtest.presentation.product.adapter;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.ninhttd.devtest.R;

public class SpCungLoaiAdapter extends RecyclerView.Adapter<SpCungLoaiAdapter.ViewHolder> {
    @NonNull
    @Override
    public SpCungLoaiAdapter.ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.view_sp_cung_loai, parent, false);
        return new SpCungLoaiAdapter.ViewHolder(view);
    }

    @Override
    public void onBindViewHolder(@NonNull SpCungLoaiAdapter.ViewHolder holder, int position) {

    }

    @Override
    public int getItemCount() {
        return 5;
    }

    public class ViewHolder extends RecyclerView.ViewHolder {
        public ViewHolder(@NonNull View itemView) {
            super(itemView);
        }
    }
}

File: app/src/main/java/com/ninhttd/devtest/presentation/product/adapter/SpCungLoaiAdapter.kt
package com.ninhttd.devtest.presentation.product.adapter

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView

import com.ninhttd.devtest.R

class SpCungLoaiAdapter : RecyclerView.Adapter<SpCungLoaiAdapter.ViewHolder>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): SpCungLoaiAdapter.ViewHolder {
        val view = LayoutInflater.from(parent.context).inflate(R.layout.view_sp_cung_loai, parent, false)
        return SpCungLoaiAdapter.ViewHolder(view)
    }

    override fun onBindViewHolder(holder: SpCungLoaiAdapter.ViewHolder, position: Int) {

    }

    override fun getItemCount(): Int {
        return 5
    }

    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView)
}
